packages/core/admin/ee/server/routes/review-workflows.js

```diff
@@ -142,6 +142,23 @@ module.exports = {
         ],
       },
     },
+    {
+      method: 'GET',
+      path: '/content-manager/(collection|single)-types/:model_uid/:id/stages',
+      handler: 'stages.listAvailableStages',
+      config: {
+        middlewares: [enableFeatureMiddleware('review-workflows')],
+        policies: [
+          'admin::isAuthenticatedAdmin',
+          {
+            name: 'admin::hasPermissions',
+            config: {
+              actions: ['admin::review-workflows.read'],
+            },
+          },
+        ],
```

Comment:
At some point before this release we need to discuss about permissions too! I dont recall what decision we took about this, but what permisson do we assign to this endpoint? we can discuss it somewhere else!